src/sdk/namespace_fs.js

```diff
@@ -2945,9 +2965,10 @@ class NamespaceFS {
                 }
                 break;
             } catch (err) {
-                dbg.warn(`NamespaceFS._delete_latest_version: Retrying retries=${retries} latest_ver_path=${latest_ver_path}`, err);
+                dbg.warn(`NamespaceFS._delete_latest_version error: retries=${retries} latest_ver_path=${latest_ver_path}`, err);
```

Comment:
We have this printing in more places; if you change it here, it will be better in all the rest as well.